### PR TITLE
[qtmozembed] Test script cleanup. Fixes JB#55182 OMP#JOLLA-292

### PR DESCRIPTION
**Pull request by mkenttala**

### CODE REVIEW

File: tests/auto/run-tests.sh [hunk 1]
```diff
@@ -1,35 +1,20 @@
...
   export QML_IMPORT_PATH=/opt/tests/qtmozembed/imports
 fi
-CURDIR=$PWD
-export QTTESTSROOT=${QTTESTSROOT:-"/opt/tests/qtmozembed"}
-if [ "$QTTESTSLOCATION" != "" ]; then
-  cd $QTTESTSLOCATION
-fi
-export QTTESTSLOCATION=${QTTESTSLOCATION:-"/opt/tests/qtmozembed/auto/mer-qt$QTVER"}
 
-#export MOZ_LOG=EmbedLiteTrace:5,EmbedNonImpl:5,EmbedLiteApp:5,EmbedLiteView:5,EmbedLiteViewThreadParent:5
```
Comment (rainemak) [inline tests/auto/run-tests.sh:24]: This was there as reminder.

File: tests/auto/run-tests.sh [hunk 2]
```diff
@@ -1,35 +1,21 @@
 #!/bin/sh
 
-QTVER=5
-
-export LANG=en_GB.UTF-8
-export EMBED_CONSOLE=stacktrace
```
Comment (rainemak) [inline tests/auto/run-tests.sh:6]: Also logging is good test cases.

**Review by rainemak — APPROVED** [review]
Please return EMBED_CONSOLE back as well. That's for logging JS output. Please rebase before merging on of top of origin/master if not already.

Approving already.

**Review by okodron — APPROVED** [review]

**Comment by rainemak** [comment]
Better would be if commit / pr title would refer to test runner or test script not tests themselves.

**Pull request merged**